### ISSUE TITLE
Can not contact author

### ISSUE THREAD

**Issue by slottybon** [MichaelHillcox/Echelon#10]
Hey there!
 
Emails I send to private@michaelhillcox.co.uk are returned with SMTP error "Recipient not found" so you will have to bear with my message inside this issue.

------
Did you know about this (kind of active) repo?
https://github.com/markweirath/echelon

I am very interested in the continued development of this project. I wrote a plugin and maybe I will make another one. Would they still be compatible with your new version?
Could you estimate when a secure use of your version is possible?

Thanks for the effort! 
So nice to know somebody still works on this.

**Comment by MichaelHillcox** [comment]
@slottybon I've ever so sorry, I have forgotten to update my email address. My new one is hello.michaelhillcox@gmail.com.

As for the plugins: I might be able to work something but plugins are currently not planned due to the fact that I will be building everything I find that is relevant and useful into the application. What sort of plugin did you make?

The repo you have linked to is not being updated anymore and the "active" dev has told me he will not be continuing.

**Comment by slottybon** [comment]
Yeah right, no problem.

The plugin is game specific for CoD4. Since Punkbuster service was no longer supported, you can  take screenshots of cheaters/hackers using a feature of the new unofficial [CoD4X](https://cod4x.me/) ([github](https://github.com/D4edalus/CoD4x_Server)) release. These screenshots contain some extra information like player GUID, player name, time taken, map and servername.
With the plugin I wrote you can manage these screenshots. It adds the screenshot and its informations to the player details page (ClientBio I think) and has seperate own plugin pages.

So I do not think the plugin is relevant for a more general version of Echelon.

**Comment by MichaelHillcox** [comment]
@slottybon I am currently reviewing the code for Echelon v2 to see how salvageable the code is. I will also at the same time review the code to see how it is implementing plugins. If it does not seem too complex and something I would be able to to implement with out messing up existing plugins then I will do my best to do so. Although it is possible that one of those things will cause issues and I will need to do it a different way essentially breaking existing plugins.

I also happen to run a CoD4 server and love your plugin idea. very nice. Although we have a different implementation of the same sort of thing.

**Comment by slottybon** [comment]
Sounds cool. 
A secure and stable base should always take priority over plugin implementation.But if you happen to get that far (although I don't know how your plugin implementation may look like) adapting the standard plugins (if necessary) should not be too hard.

Thanks for the effort one more time.
I know this takes up an awfull lot of sparetime.

**Comment by MichaelHillcox** [comment]
@slottybon If your a php programmer too. Feel free to fork the code and add your own additions. I am more than happy to have help on this project.

Also addon support will be likely after release stage. I can add it as a planned feature :)

**Comment by efinst0rm** [comment]
Hey,

Would guys share your cod4 screen shot plugin if that's not an issue?

Thank you

**Comment by slottybon** [comment]
@MichaelHillcox Yes, maybe I will do that If I got some free time left. Addons/Plugins would be a great thing to have.

@efinst0rm Right now I do not think you could use it as is. You see the plugin was made to fit into my clans environment. Maybe I will make a more general version of the plugin public after upgrading to Echelon v3.
